tests/src/Interop/RefCharArray/RefCharArrayNative.cpp

```diff
@@ -6,7 +6,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 
-size_t LEN = 10;
+const int LEN = 10;
```

Comment:
All `for` loops below use `int` rather than `size_t` and that generated signed/unsigned mismatch warnings. This gets converted to `size_t` when calling `CoTaskMemAlloc` but since it is `const` the compiler can figure out that it is not negative and doesn't issue a warning.